src/managementgroup/azext_managementgroup/generated/_help.py

```diff
@@ -0,0 +1,241 @@
+        text: |-
+               az managementgroup management-group show --expand "children" --recurse true --group-id \
+"20000000-0001-0000-0000-000000000000"
+"""
+
+helps['managementgroup management-group create'] = """
+    type: command
+    short-summary: "Create a management group. If a management group is already created and a subsequent create \
+request is issued with different properties, the management group properties will be updated."
+    examples:
+      - name: PutManagementGroup
+        text: |-
+               az managementgroup management-group create --display-name "ChildGroup" --id \
+"/providers/Microsoft.Management/managementGroups/RootGroup" --group-id "ChildGroup"
+"""
+
+helps['managementgroup management-group update'] = """
+    type: command
+    short-summary: "Update a management group."
+    examples:
+      - name: PatchManagementGroup
+        text: |-
+               az managementgroup management-group update --group-id "ChildGroup" --display-name \
+"AlternateDisplayName" --parent-group-id "/providers/Microsoft.Management/managementGroups/AlternateRootGroup"
+"""
+
+helps['managementgroup management-group delete'] = """
+    type: command
+    short-summary: "Delete management group. If a management group contains child resources, the request will fail."
+    examples:
+      - name: DeleteManagementGroup
+        text: |-
+               az managementgroup management-group delete --group-id "GroupToDelete"
+"""
+
+helps['managementgroup management-group show-descendant'] = """
+    type: command
+    short-summary: "List all entities that descend from a management group."
+    examples:
+      - name: GetDescendants
+        text: |-
+               az managementgroup management-group show-descendant --group-id "20000000-0000-0000-0000-000000000000"
+"""
+
+helps['managementgroup management-group wait'] = """
+    type: command
+    short-summary: Place the CLI in a waiting state until a condition of the managementgroup management-group is met.
+    examples:
+      - name: Pause executing next line of CLI script until the managementgroup management-group is successfully \
+created.
+        text: |-
+               az managementgroup management-group wait --expand "children" --recurse true --group-id \
+"20000000-0001-0000-0000-000000000000" --created
+      - name: Pause executing next line of CLI script until the managementgroup management-group is successfully \
+deleted.
+        text: |-
+               az managementgroup management-group wait --expand "children" --recurse true --group-id \
+"20000000-0001-0000-0000-000000000000" --deleted
+"""
+
+helps['managementgroup management-group-subscription'] = """
+    type: group
+    short-summary: Manage management group subscription with managementgroup
+"""
+
+helps['managementgroup management-group-subscription create'] = """
+    type: command
+    short-summary: "Associates existing subscription with the management group."
```

Comment:
Please use the activate voice
```suggestion
    short-summary: "Associate existing subscription with the management group."
```